.github/workflows/build_and_release.yml

```
@@ -0,0 +1,78 @@
+          npm i -g @vercel/ncc
+          npm i @actions/github
+          npm run build
+      - name: Commit
+        run: |
+          git config --global user.name 'paradisemom'
+          git config --global user.email '142639231+paradisemom@users.noreply.github.com'
+          git commit -am "dist build"
+          git push
+
+  test:
+    runs-on: ubuntu-lastest
```

actionlint

label "ubuntu-lastest" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2022", "windows-2019", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-22.04", "ubuntu-20.04", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-14.0", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "macos-13.0", "macos-12-xl", "macos-12-xlarge", "macos-12-large", "macos-12", "macos-12.0", "macos-11", "macos-11.0", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)